wiki/History_of_osu!/2014/en.md

```diff
@@ -0,0 +1,27 @@
+# 2014
+
+*Main page: [History of osu!](/wiki/HOO).*
+
+## March
+
+As skinning elements expanded, the element, `playfield.jpg` was removed in it's line of duty and replaced with a new generic one that was unskinnable. `playfield.jpg` was an element that allowed you to use any image as the default background of a beatmap, if one isn't supplied.
```

Comment:
new generic one that **is** unskinnable

Reply:
can you explain why it should be `is` instead of `was`?

Reply:
because it's still unskinable, it didnt change

Reply:
I will change it, but I don't fully agree to it :s